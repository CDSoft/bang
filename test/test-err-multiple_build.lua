generator(false)

rule "foo" { command = true }
rule "bar" { command = true }

build "bar" { "foo" }
build "baz" { "foo" }
build "bar" { "foo" } -- same definition
build "bar" { "bar" } -- [test/test-err-multiple_build.lua:7] ERROR: build bar: multiple definition
