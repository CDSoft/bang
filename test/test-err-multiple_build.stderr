[test/test-err-multiple_build.lua:9] ERROR: build bar: multiple definition
